### PR TITLE
feat: support parsing and providing Pulumi outputs as secret

### PR DESCRIPTION
**Pull request by rawkode**

### CODE REVIEW

File: pkg/universe.dagger.io/alpha/pulumi/up.sh [hunk 1]
```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env bash
-set -xeo pipefail
+set -x
```
[the remark this answers is not in the document]
Comment (rawkode) [reply pkg/universe.dagger.io/alpha/pulumi/up.sh:2]: It was redundant as `bash.#Run` runs `bash —norc -eo pipefail <script>`

[the remark this answers is not in the document]
Comment (rawkode) [reply pkg/universe.dagger.io/alpha/pulumi/up.sh:2]: Unless we need to provide it on both? Then I’ll revert. I could be misunderstanding

[the remark this answers is not in the document]
Comment (rawkode) [reply pkg/universe.dagger.io/alpha/pulumi/up.sh]: Signed-off and reverted. If it's redundant, we can clean later and document the expectations of scripts within packages.

**Comment by gerhard** [comment]
As soon as the checks pass, this is going in, just in time for `v0.2.22`, which was delayed by a few days due to https://github.com/dagger/dagger/pull/2749.

**Review by gerhard — APPROVED** [review]
👍

**Comment by rawkode** [comment]
Example usage for anyone that finds this PR:

https://github.com/rawkode/rawkode/blob/main/projects/web-links/build.cue

**Comment by gerhard** [comment]
Now available in https://github.com/dagger/dagger/releases/tag/v0.2.22

**Pull request merged**